pom.xml

```diff
@@ -7,16 +7,17 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.0.6</version>
+    <version>4.0.1</version>
   </parent>
 
   <artifactId>quibble</artifactId>
   <version>1.0.1-SNAPSHOT</version>
   <name>Quibble</name>
 
   <scm>
-    <connection>scm:git:ssh://git@stash:7999/hdw/hww-expat.git</connection>
-    <developerConnection>scm:git:ssh://git@stash:7999/hdw/hww-expat.git</developerConnection>
+    <connection>scm:git:git@github.com:HotelsDotCom/quibble.git</connection>
```

Comment:
Don't know how this ever got pushed as it was TBH, anyway....

Reply:
good find def need to be fixed